### ISSUE TITLE
warp10-quantumviz broken in Firefox

### ISSUE THREAD

**Issue by babolivier** [cityzendata/warp10-quantumviz#58]
Here's a graph in Firefox:

![capture d ecran de 2017-12-04 17-19-04](https://user-images.githubusercontent.com/5547783/33563066-41aa1156-d917-11e7-9785-e45598b68a25.png)

And here's the same graph in Chromium:

![capture d ecran de 2017-12-04 17-13-49](https://user-images.githubusercontent.com/5547783/33562933-dea2a9ec-d916-11e7-8a6c-e4a1ef227965.png)

The code is:

```html
<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8">
        <title>Infra dashboard</title>
        <script src="https://cdn.cityzendata.net/quantumviz/latest/webcomponentsjs/webcomponents-lite.js"></script>
        <link rel="import" href="https://cdn.cityzendata.net/quantumviz/latest/warp10-quantumviz/warp10-quantumviz.html">
    </head>
    <body>
        <warp10-quantumviz host="https://warp10.gra1.metrics.ovh.net/api/v0">
            [My WarpScript]
        </warp10-quantumviz>
    </body>
</html>

```

This bug has been witnessed just now on Firefox Quantum (57.0.1, GNU/Linux) with a blank profile.

**Comment by LostInBrittany** [comment]
OK, I've just replicated the bug, going to deal with it

**Comment by LostInBrittany** [comment]
Fixed on 2.1.0-b6. There is still a problem with c3 implementation, we will need to see it with @Giwi but for the moment I revert to DyGraphs implementation.

**Comment by babolivier** [comment]
Awesome, thanks! 😁
